examples/classifier.c

```diff
@@ -462,34 +492,38 @@ void get_predictions (network* net, char** paths, char** labels, int m, int clas
     args.type = OLD_CLASSIFICATION_DATA;
 
     pthread_t load_thread = load_data_in_thread(args);
+    _Bool hasRemainder = 1;
     for(curr_batch = net_batch; curr_batch < max; curr_batch += net_batch)
```

Comment:
You should use **net->batch** instead of **net_batch** in the header of this loop since the update of **net->batch** doesn't update the value of **net_batch** in here as well. It would also be nice to have the name of **net_batch** changed to something like _original_value_ since it does represent its role in here better.

Reply:
Well spotted! Fixed in my last commit.